frontend/src/components/common/FlexBox/FlexBox.stories.tsx

```diff
@@ -0,0 +1,172 @@
+        type: 'select',
+      },
+      description: '플렉스 컨테이너 안에 있는 아이템의 위치를 조절할 수 있습니다.',
+    },
+    noRadius: {
+      options: { none: false, all: 'all', top: 'top', bottom: 'bottom' },
+      control: {
+        type: 'select',
+      },
+      description: '특정 방향의 radius 속성을 제거할 수 있습니다.',
+    },
+    outlined: {
+      control: {
+        type: 'boolean',
+      },
+      description: 'true: 플렉스 컨테이너에 검은색 테두리가 생깁니다.',
+    },
+    background: {
+      control: {
+        type: 'color',
+      },
+      description: '선택한 색상에 따라 플렉스 컨테이너의 배경색이 변합니다.',
+    },
+    direction: {
+      description: `row: 플렉스 컨테이너 안 아이템이 수직 방향으로 정렬됩니다. 기본값입니다.<br>column: 플렉스 컨테이너 안 아이템이 수직 방향으로 정렬됩니다.`,
+    },
+    nowrap: {
+      description:
+        'true: 플렉스 컨테이너, 아이템 크기에 상관없이 무조건 한 줄로 정렬합니다. 아이템의 width, height가 정해져 있지 않다면 플렉스 컨테이너의 크기만큼 늘어납니다.',
+    },
+    gap: {
+      description:
+        '숫자를 입력해 플렉스 컨테이너 안 아이템 간의 간격을 조절할 수 있습니다. `입력한 숫자 x 4px` 만큼 간격이 늘어납니다.',
+    },
+    rowGap: {
+      description:
+        '숫자를 입력해 플렉스 컨테이너 안 아이템 간의 행 높이를 조절할 수 있습니다. `입력한 숫자 x 4px` 만큼 간격이 늘어납니다.',
+    },
+    columnGap: {
+      description:
+        '숫자를 입력해 플렉스 컨테이너 안 아이템 간의 열 너비를 조절할 수 있습니다. `입력한 숫자 x 4px` 만큼 간격이 늘어납니다.',
+    },
+    children: {
+      table: {
+        disable: true,
+      },
+    },
+    css: {
+      description: '원하는 css를 적용할 수 있습니다.',
+    },
+  },
+} satisfies Meta<typeof FlexBox>;
+
+export default meta;
+
+export const Default = (args: FlexBoxProps) => {
+  return <FlexBox {...args} />;
+};
+
+interface SizeProps {
+  width?: number;
+  height?: number;
+}
+
+const size = ({ width, height }: SizeProps): CSSProp => css`
+  width: ${width}rem;
+  height: ${height}rem;
+`;
+
+export const Positions = () => {
+  return (
+    <FlexBox nowrap columnGap={4} position="between">
+      <FlexBox direction="column" rowGap={5}>
+        <FlexBox outlined position="start" css={size({ height: 10 })}>
+          {Boxes()}
+        </FlexBox>
+        <FlexBox outlined position="center" css={size({ height: 10 })}>
+          {Boxes()}
+        </FlexBox>
+        <FlexBox outlined position="end" css={size({ height: 10 })}>
+          {Boxes()}
+        </FlexBox>
+        <FlexBox outlined position="between" css={size({ height: 10 })}>
+          {Boxes()}
+        </FlexBox>
+      </FlexBox>
+      <FlexBox outlined direction="column" position="start" css={size({ width: 12 })}>
+        {Boxes()}
+      </FlexBox>
+      <FlexBox outlined direction="column" position="center" css={size({ width: 12 })}>
+        {Boxes()}
+      </FlexBox>
+      <FlexBox outlined direction="column" position="end" css={size({ width: 12 })}>
+        {Boxes()}
+      </FlexBox>
+      <FlexBox outlined direction="column" position="between" css={size({ width: 12 })}>
+        {Boxes()}
+      </FlexBox>
+    </FlexBox>
+  );
+};
+
+export const Gap = () => {
+  return (
+    <Container>
+      <FlexBox position="center" rowGap={4} columnGap={8}>
```

Comment:
```suggestion
      <FlexBox alignItems={} justifyContent={} rowGap={4} columnGap={8}>
```
원래 flex를 사용할 때와 유사한 방식이 더 좋을 것 같습니다!